tests/test_dataflow.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
```

Comment:
You're so cruel! Now I have to remind it. It has something to do with the `swf` module iirc.